### PR TITLE
 fixed search for argument types int, float and bool

### PR DESCRIPTION
**Pull request by RushabhDalwadi**

### CODE REVIEW

File: backend/AsterixUtils.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def executePeterListQuery(query):
 def queryAsterix(query):
 	response = executePeterListQuery(query)
 	if response != None:
-		return json.dumps(json.loads(response.text)['results'])
+		return json.dumps(json.loads(response.text))#['results'])
```
Comment (chriswongtv) [inline backend/AsterixUtils.py:27]: Is this change intentional?

File: backend/app.py [hunk 2]
```diff
@@ -15,17 +16,20 @@
 			template_folder = "../dist")
 cors = CORS(app, resources={r"/api/*": {"origins": "*"}})
 
-DB = 1 # 1 = AsterixDB, 0 = Couchbase
-
-@app.route('/', defaults={'path': ''})
-@app.route('/<path:path>')
-def catch_all(path):
-	return render_template("index.html")
+#@app.route('/', defaults={'path': ''})
```
Comment (chriswongtv) [inline backend/app.py]: I believe this would break the front-end by not serving it.

**Review by chriswongtv — CHANGES_REQUESTED** [review]

File: backend/AsterixUtils.py (hunk 1 above)
Comment (RushabhDalwadi) [reply backend/AsterixUtils.py:27]: Yes. I tested other parts of the code that used it function.

File: backend/app.py (hunk 2 above)
Comment (RushabhDalwadi) [reply backend/app.py]: Oh ok. I don't completely understand what it does. I commented it out because for some reason the "/api/search" and "/api/subscribe" stopped working, and commenting this out fixed it.
How is the front-end using it?

Comment (chriswongtv) [reply backend/app.py]: It serves the front-end code that's been built to the client. Did you run `npm run build` to build the front-end prior to running it? Otherwise, you might not have the `dist` folder as it is excluded in the `.gitignore`.

Comment (RushabhDalwadi) [reply backend/app.py]: Nevermind. It works now.

Comment (chriswongtv) [reply backend/app.py]: Awesome, thanks!

**Review by chriswongtv — APPROVED** [review]
